TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -20,11 +20,27 @@
 import Foundation
 
 protocol PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
-                    identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock)
-  func verifyFactor(withSid sid: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock)
-  func updateFactor(withSid sid: String, withPushToken pushToken: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock)
-  func deleteFactor(withSid sid: String, success: @escaping EmptySuccessBlock, failure: @escaping TwilioVerifyErrorBlock)
+  func createFactor(withAccessToken accessToken: String,
+                    friendlyName: String,
+                    pushToken: String?,
```

Comment:
As this is an internal class, we could move this to be the latest parameter before the success and failure block, wdyt?

Reply:
Sounds good! I just did it on the last pushed commit